.github/workflows/quiz_app_ci.yml

```diff
@@ -0,0 +1,49 @@
+name: QuizApp CI
```

Comment:
I would also parametrize the job, so it can by run manually.
F.e. two boolean parameters: `test` and `lint`, true as default.
See: workflow_dispatch -> inputs.

Note: Dispatching the job manually might work only on main branch, it's silly like that.